lib/whitehall_importer.rb

```diff
@@ -23,9 +25,26 @@ def self.create_migration(organisation_content_id, document_type)
     end
   end
 
-  def self.import_and_sync(whitehall_document)
-    whitehall_import = WhitehallMigration::DocumentImport.create!(
-      whitehall_document_id: whitehall_document["id"],
+  def self.queue_migration_jobs(whitehall_migration)
+    documents = WhitehallMigration::DocumentImport.where(whitehall_migration_id: whitehall_migration.id)
+    documents.each do |document|
```

Comment:
Looks like there should be an ActiveRecord association set up for this so that we can do `whitehall_migration.documents.each`

Reply:
We've created an association for this and updated the relevant part of the import code.